source/features/index.tsx

```diff
@@ -150,6 +157,29 @@ const setupPageLoad = async (id: FeatureID, config: InternalRunConfig): Promise<
 	}
 };
 
+// Fetch hotfix releases asynchronously
+// every 6 hours ( 4 times a day )
+// this is achieved by GETting a JSON file
+// from the hotfix branch and comparing
+// with the current version of extension
```

Comment:
These comments are rephrasing the whole function line by line, that's not useful. Comments should either explain what's going on if code isn't immediately clear or _why_ something is being done.

Reply:
```suggestion
```